vispy/scene/cameras/base_camera.py

```diff
@@ -108,6 +109,19 @@ def depth_value(self, value):
             raise ValueError('depth value must be positive')
         self._depth_value = value
 
+    @property
```

Comment:
Yeah, I think we currently try to do that, and @jdreaver had a PR at one point that fixed it but our changes from the sprint made it impossible to merge. So this is more of a stop-gap measure, although I could still see applications where you might want your mouse movement effects to be magnified by some factor, so I don't think it would become totally obsolete when we eventually have a better / more complete solution in place.